docs/gyakorlat/rest/index.md

```diff
@@ -21,39 +21,39 @@ Amit érdemes átnézned:
 
 ## Gyakorlat menete
 
-A gyakorlat végig vezetett, a gyakorlatvezető utasításai szerint haladjunk. Egy-egy részfeladatot próbáljunk meg először önállóan megoldani, utána beszéljük meg a megoldást közösen. Az utolsó feladat opcionális, ha belefér az időbe.
+A gyakorlat végig vezetett, a gyakorlatvezető utasításai szerint haladjunk. Egy-egy részfeladatot próbáljunk meg először önállóan megoldani, utána beszéljük meg a megoldást közösen. Az utolsó és utolsó előtti feladat opcionális, ha belefér az időbe.
 
 !!! info ""
     Emlékeztetőként a megoldások is megtalálhatóak az útmutatóban is. Előbb azonban próbáljuk magunk megoldani a feladatot!
 
-## Feladat 0: Adatbázis létrehozása, ellenőrzése
+## 0. Feladat: Adatbázis létrehozása, ellenőrzése
 
 Az adatbázis az adott géphez kötött, ezért nem biztos, hogy a korábban létrehozott adatbázis most is létezik. Ezért először ellenőrizzük, és ha nem találjuk, akkor hozzuk létre újra az adatbázist. (Ennek mikéntjét lásd az [első gyakorlat anyagában](../transactions/index.md).)
 
-## Feladat 1: Projekt megnyitása
+## 1. Feladat: Projekt megnyitása
 
 1. Töltsük le a méréshez tartozó projekt vázat!
 
     - Nyissunk egy _command prompt_-ot
     - Navigáljunk el egy tetszőleges mappába, például `c:\work\NEPTUN`
     - Adjuk ki a következő parancsot: `git clone --depth 1 https://github.com/bmeviauac01/gyakorlat-rest-kiindulo.git`
 
-1. Nyissuk meg a `rest` könyvtár alatti _sln_ fájlt Visual Studio-val.
+1. Nyissuk meg a leklónozott könyvtár alatti _sln_ fájlt Visual Studio-val.
 
 1. Vizsgáljuk meg a projektet.
 
     - Ez egy ASP.NET Core Web API projekt. Kifejezetten REST API-k kiszolgálásához készült. Ha F5-tel elindítjuk, akkor magában tartalmaz egy webszervert a kérések kiszolgálásához.
-    - Nézzük meg a `Program.cs` tartalmát. Nem kell értenünk, hogy mi történik itt pontosan, csak lássuk, hogy ez olyan, mint egy konzol alkalmazás: a `Main` függvényben elindít egy webszervert.
-    - Az adatbázisunk Entity Framework leképzése (_Code First_ modellel) megtalálható a `Dal` mappában. Az `DataDrivenDbContext` lesz az elérés központi osztálya. A _connection stringet_ javítsuk ki szükség esetén ebben az osztályban az `OnConfiguring` függvényben.
-
-        !!! note ""
-            A connection stringet természetesen nem célszerű beégetni a forráskódba. Mi ezt csupán egyszerűsítésként alkalmazzuk.
-
+    - Nézzük meg a `Program.cs` tartalmát. Lényegében két részből áll:
+      - Létrehoz egy `WebApplicationBuilder` objektumot, amelynek a `Services` tulajdonságán keresztül tudjuk konfigurálni a Dependency Injection konténert.
```

Comment:
A futó "mkdocs" környezetben nincs pluszban behúzva, hanem a szülővel egy szinten jelenik meg.

Reply:
ez is jogos, mert a vscode csak 2 space-t rakott ki 4 helyett